src/protocols/ethereum/helpers/index.ts

```diff
@@ -1,9 +1,14 @@
 import { fromWei, toChecksumAddress, toWei } from 'web3-utils';
+import { FMT_BYTES, FMT_NUMBER } from 'web3-types';
 import { getBlock, getTransaction } from 'web3-eth';
+import Contract from 'web3-eth-contract';
 import BigNumber from 'bignumber.js';
+
 import { AccountAddress, ITransaction } from '@/types';
 import { PROTOCOLS } from '@/constants';
-import { ETH_CONTRACT_ID } from '../config';
+import { bigIntToHex } from 'web3-eth-accounts';
+import { ERC20_ABI, ETH_CONTRACT_ID } from '../config';
+import { useEthNetworkSettings } from '../composables/ethNetworkSettings';
```

Comment:
Helper functions should not import composables as it breaks the hierarchy:
Constants -> Utils -> Libs -> Composables -> Components